x11/font-spleen/Portfile

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                font-spleen
+github.setup        fcambus spleen 1.9.1
+github.tarball_from releases
+revision            0
+categories          x11 x11-font graphics fonts
```

Comment:
I don't think `x11-font` and `fonts` are categories that are present in MacPorts, let's not add them

Reply:
Both exist, and `fonts` is pretty wildly used:
```shell
❯ egrep -rn '^categories.*fonts.*'
./devel/t1lib/Portfile:8:categories          devel graphics fonts
./x11/freefont-ttf/Portfile:7:categories          x11 fonts
./x11/sazanami-font/Portfile:6:categories        x11 fonts
./x11/urw-core35-fonts/Portfile:8:categories          x11 fonts
./x11/ttf-arphic-uming/Portfile:5:categories      x11 fonts
./x11/urw-fonts/Portfile:7:categories      x11 fonts
./x11/mikachanfont/Portfile:6:categories     x11 fonts
./x11/kochi-substitute/Portfile:6:categories         x11 fonts
./x11/ttf-arphic-ukai/Portfile:5:categories      x11 fonts
./x11/liberation-fonts/Portfile:16:categories          x11 fonts
./x11/monafont/Portfile:10:categories      x11 fonts
./x11/efont-unicode/Portfile:6:categories     x11 fonts
./x11/mplus-fonts/Portfile:11:categories                x11 fonts
./x11/dejavu-fonts/Portfile:6:categories          x11 fonts
./x11/font-spleen/Portfile:10:categories          x11 x11-font graphics fonts
./graphics/ttfautohint/Portfile:14:categories         graphics fonts
./graphics/libwmf/Portfile:8:categories          graphics fonts
./graphics/t1utils/Portfile:7:categories      graphics print fonts
./graphics/fontforge/Portfile:13:categories          graphics fonts
./graphics/fntsample/Portfile:9:categories          graphics fonts print
```
and
```shell
❯ egrep -rn '^categories.*x11-font.*'
./x11/font-misc-ethiopic/Portfile:8:categories          x11 x11-font graphics
./x11/xorg-encodings/Portfile:9:categories          x11 x11-font graphics
./x11/font-alias/Portfile:7:categories          x11 x11-font graphics
./x11/font-spleen/Portfile:10:categories          x11 x11-font graphics fonts
```

Still want me to drop them?

Reply:
ah okay, my understanding is/was that the categories should correspond to the the "subdirectories" in the MacPorts' tree. But if others use things like `font`, `x11-font` in addition to those then it's fine with me.